setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup, find_packages
+from codecs import open
+from os import path
+
+import progresso
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='progresso',
+    version=progresso.__version__,
```

Comment:
> it might be a problem to reference the module from the setup.py.

I think it's safe while you don't have any third party dependency, but I may be missing something :)

> Instead, it's recommended to use pkg_info to get the version as follows:

Very interesting! I'm testing that right away :)